fazel_file_generator/fazel_file_creator.py

```diff
@@ -154,11 +155,12 @@ def main(args=None):
             if not args.quiet:
                 print("Source not found in common list, searching online")
             source = SkyCoord.from_name(args.source)
-            source_found = False
+            source_found = True
         except Exception:
             try:
                 source = SkyCoord.from_name("PSR " + args.source)
                 source_name = "psr" + source_name
+                source_found = True
```

Comment:
```suggestion
                source_found = False
```

Reply:
Same here, I think this line is not needed.